internal/internal_coroutines_test.go

```diff
@@ -551,6 +551,191 @@ func TestBlockingSelect(t *testing.T) {
 	require.EqualValues(t, expected, history)
 }
 
+func TestSelectBlockingDefault(t *testing.T) {
```

Comment:
We probably don't want the same test both here and in `internal_workflow_testsuite_test.go` right? Which file should hold this test?

Reply:
Actually maybe this is fine, one set tests w/ and w/o flags, and the other does default behavior